### PR TITLE
add tts chunk size

### PR DESCRIPTION
**Pull request by ali-ziaei**

### CODE REVIEW

File: vocode/streaming/streaming_conversation.py [hunk 1]
```diff
@@ -368,7 +367,7 @@ async def process(
                     message.text,
                     synthesis_result,
                     item.interruption_event,
-                    TEXT_TO_SPEECH_CHUNK_SIZE_SECONDS,
```
Comment (adam-earle) [inline vocode/streaming/streaming_conversation.py:371]: We'll still have this (now unused?) CONSTANT floating around... not a big deal, probably fine as is.

**Review by adam-earle — APPROVED** [review]
lgtm, but review is lite based on previous deep dive.

File: vocode/streaming/models/synthesizer.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ class SynthesizerConfig(TypedModel, type=SynthesizerType.BASE.value):
     audio_encoding: AudioEncoding
     should_encode_as_wav: bool = False
     sentiment_config: Optional[SentimentConfig] = None
+    text_to_speech_chunk_size_seconds: float = 1.0
```
Comment (adam-earle) [inline vocode/streaming/models/synthesizer.py:48]: If we know we want a default of <1 here I think we can just include it in this PR?

Comment (ali-ziaei) [reply vocode/streaming/models/synthesizer.py:48]: offline discussed: It is on other PR

File: vocode/streaming/streaming_conversation.py (hunk 1 above)
Comment (ali-ziaei) [reply vocode/streaming/streaming_conversation.py:371]: good catch

**Pull request merged**